File: CobongMemo/app/src/main/java/cobong/jeongwoojin/cobongmemo/cobongmemo/view/memo/voicememo/VoiceNavigator.java
package cobong.jeongwoojin.cobongmemo.cobongmemo.view.memo.voicememo;

public interface VoiceNavigator {
    default public void onRecordClick() {};
    default public void onStopClick() {};
    default public void onExitClick() {};
    default public void onPlayClick() {};
    default public void onPauseClick() {};
}

File: CobongMemo/app/src/main/java/cobong/jeongwoojin/cobongmemo/cobongmemo/view/memo/voicememo/VoiceNavigator.kt
package cobong.jeongwoojin.cobongmemo.cobongmemo.view.memo.voicememo

interface VoiceNavigator {
    open fun onRecordClick() {}
    open fun onStopClick() {}
    open fun onExitClick() {}
    open fun onPlayClick() {}
    open fun onPauseClick() {}
}
